pokemon-minisite/src/environment.ts

```diff
@@ -0,0 +1,6 @@
+export const environment = {
+    production: false,
+    apiUrl: 'https://pokeapi.co/api/v2/pokemon/',
+    imageUrl: 'https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/',
```

Comment:
enter the image and verify mail to params object

Reply:
done